pkg/operator/ceph/object/user/controller_test.go

```diff
@@ -52,7 +52,7 @@ const (
 	"display_name": "my-user",
 	"email": "",
 	"suspended": 0,
-	"max_buckets": 1000,
```

Comment:
In addition to these unit tests, how about if we enable the new settings in the test object user for the integration tests [here](https://github.com/rook/rook/blob/993b82674ff7cc7a1bc64cadb9f0dacec5bf91d7/tests/framework/installer/ceph_manifests.go#L441)?